packages/front-end/pages/experiments/index.tsx

```diff
@@ -453,9 +455,14 @@ const ExperimentsPage = (): React.ReactElement => {
                 <thead>
                   <tr>
                     <th></th>
-                    <SortableTH field="name" className="w-100">
+                    <SortableTH field="name" className="nowrap">
                       Experiment
                     </SortableTH>
+                    {hasDescription && (
+                      <SortableTH field="description" className="">
+                        Description
+                      </SortableTH>
+                    )}
```

Comment:
I don't think this column should be shown by default.  The table is already really busy.  What if we add a cog icon to configure the table so users can add in additional columns.  Would get us pretty close to a full advanced search without cluttering the UI for others.